lib/vagrant-dns/command.rb

```diff
@@ -93,8 +93,18 @@ def manage_installation(vms, options)
         elsif options[:purge]
           installer.purge!
         end
+      elsif Vagrant::Util::Platform.linux?
+        installer_options = options.fetch(:installer_opts, {})
+        installer = VagrantDNS::Installers::Linux.new(tmp_path, installer_options)
+
+        if options[:install]
```

Comment:
Ah! That's true. I implemented `purge!` initially but removed it since I had it only call out to `uninstall!`. Thinking about it some more now (without the code in front of me) I think the `uninstall!` action doesn't remove the config file under `.vagrant.d/` so I should do that regardless of the cleaner code you proposed!

Reply:
Should be fixed now! (If you approve, please let me squash the `!fixup` first!).